packages/viz/src/lib/style/helpers/ramps.ts

```diff
@@ -0,0 +1,34 @@
+import {CartoStylingError, stylingErrorTypes} from '../../errors/styling-error';
+
+export function numericRamp(featureName: string, numericValues = [], colors = []) {
+  if (!featureName) {
+    throw new CartoStylingError('Feature name is missing', stylingErrorTypes.PROPERTY_MISSING);
+  }
+
+  const lengthMismatch = numericValues.length !== colors.length;
```

Comment:
Depending on open / closed intervals, we might want to compare to colors +/- 1

I would start with closed intervals on right sides, so 0-100-500 would be useful to define 2 buckets (2 colors) like:
- (a) 0 to 100 (>=0 to <100)
- (b) 100 to 500 (>=100 to <500)

So 0 will be included in (a), 100 in (b) and 500 outside of intervals. To assign 500 a color, a new interval would be needed (eg 0, 100, 500, 1000).

How do you see this approach?

Reply:
I thought of doing that at first because it seemed more intuitive for me, but programming it the other way was easier and TurboCARTO did it with open intervals.

But I think it's better with closed intervals.